tracetesting/transactions/tests/http/test-create.yaml

```diff
@@ -0,0 +1,65 @@
+type: Test
+spec:
+  id: VZvc1g44g
+  name: Test Create
+  trigger:
+    type: http
+    httpRequest:
+      url: "${env:TARGET_URL}/api/tests"
+      method: POST
+      headers:
+      - key: Content-Type
+        value: application/json
+      body: |
+        {
+          "name": "Pokemon - List - Get a Pokemon",
+          "serviceUnderTest": {
+            "triggerType": "http",
+            "triggerSettings": {
+              "http": {
+                "url": "${env:DEMO_APP_URL}/pokemon?take=20&skip=0",
+                "method": "GET",
+                "headers": [
+                  {
+                    "key": "Content-Type",
+                    "value": "application/json"
+                  }
+                ]
+              }
+            }
+          },
+          "specs": {
+            "specs": [
+              {
+                "selector": {
+                  "query": "span[name = \"pg.query:SELECT\"]"
+                },
+                "assertions": [
+                  {
+                    "attribute": "tracetest.selected_spans.count",
+                    "comparator": ">",
+                    "expected": "0"
+                  }
+                ]
+              }
+            ]
+          }
+        }
+
+  specs:
+  - selector: span[name = "Tracetest trigger"]
+    assertions:
+      - tracetest.selected_spans.count = 1
+      - tracetest.response.status = 200
+  - selector: span[name="POST /api/tests" tracetest.span.type="http"]
+    assertions:
+    - tracetest.selected_spans.count = 1
+  - selector: span[name = "exec INSERT"]
+    assertions:
+    - tracetest.selected_spans.count = 1
+  - selector: span[name = "exec INSERT"]:first
+    assertions:
+    - sql.query contains "INSERT INTO tests"
+
+  outputs:
+    CREATED_TEST_ID: "${attr:tracetest.request.body | json_path '.id' }"
```

Comment:
this is where the `filter` concept becomes really useful. When setting outputs, you need to set it to a string value. Right now, we don't have a way to extract a single value from a json, so we cannot do this. 

Using the `json_path` hypothetical filter, you could extract any value from a json response. We could provide other filters, like `substring` for example.

Reply:
We have added a regexp field in the front end to have at least some basic support for users. It can be seen in this loom video 

https://www.loom.com/share/a57ef209f68a4e20888e8a39559affec

Reply:
I don't think regex is enough to handle complex cases. We could totally support a `regex` filter, like `${attr:tracetest.request.body | json_path '.id' | regex "\d+" }`  (this would get the ID from the response body, and from that value, apply a regex to get only the numeric parts).

But we need to have a way to get very specific outputs. If we need to rely on regexp to parse complex outputs, it won't work.

Also, what if the response is not json/text, but protobuf encoded? that's a binary format, and regex won't work there